code/game/objects/items/devices/instruments.dm

```diff
@@ -268,3 +268,38 @@
 	item_state = "electronic"
 	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+
+/obj/item/musicbeacon
+	name = "express delivery beacon"
+	desc = "Summon your tool of art."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-red"
+	item_state = "radio"
+	var/static/list/instruments
+	var/static/list/display_names = list()
+/obj/item/musicbeacon/attack_self(mob/user)
+	beacon_music(user)
+
+/obj/item/musicbeacon/proc/beacon_music(mob/M)
+	if(!instruments)

```

Comment:
This block of code is for creating display_names not instruments. instruments is an intermediate var that immediately becomes worthless when display_names is finished. instruments should be defined inside this block and check display_names instead here.